<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>examine_window</class>
 <widget class="QMainWindow" name="examine_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sát hạch lái xe bằng A1</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/icons/app_icon.ico</normaloff>assets/icons/app_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Bạn đang làm câu</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="question_order">
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>25</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Thời gian còn lại:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="clock_minute">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="clock_second">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="question">
           <property name="text">
            <string>Câu hỏi</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="image_question">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Đáp án</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QRadioButton" name="first_answer">
              <property name="text">
               <string>A</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="second_answer">
              <property name="text">
               <string>B</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="third_answer">
              <property name="text">
               <string>C</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="fourth_answer">
              <property name="text">
               <string>D</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="previous_question">
             <property name="toolTip">
              <string>Câu trước đó</string>
             </property>
             <property name="text">
              <string>Câu trước</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icons/previous.png</normaloff>../icons/previous.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="submit">
             <property name="toolTip">
              <string>Kết thúc làm bài và nộp bài</string>
             </property>
             <property name="text">
              <string>Nộp bài</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icons/submit.png</normaloff>../icons/submit.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="next_question">
             <property name="toolTip">
              <string>Câu tiếp theo</string>
             </property>
             <property name="text">
              <string>Câu tiếp</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icons/next.png</normaloff>../icons/next.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>310</width>
        <height>280</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>360</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Lưu ý&lt;/span&gt;: Các câu trả lời rồi sẽ hiển thị màu xanh lá.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Các câu hỏi</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Bạn còn</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="not_answer_question">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>25</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>câu chưa hoàn thành</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="2">
             <widget class="QPushButton" name="question_13">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 13</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="question_12">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 12</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="question_8">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 8</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QPushButton" name="question_19">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 19</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QPushButton" name="question_10">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 10</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="question_9">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 9</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="question_4">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 4</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="question_7">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 7</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="question_16">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 16</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="question_5">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 5</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="question_17">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 17</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="question_18">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 18</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="question_6">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 6</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="question_11">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 11</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="question_14">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 14</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="question_1">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 1</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="question_3">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 3</string>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QPushButton" name="question_20">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 20</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="question_2">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 2</string>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QPushButton" name="question_15">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 15</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="question_21">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 21</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="question_22">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 22</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QPushButton" name="question_23">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 23</string>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QPushButton" name="question_24">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 24</string>
              </property>
             </widget>
            </item>
            <item row="4" column="4">
             <widget class="QPushButton" name="question_25">
              <property name="styleSheet">
               <string notr="true">QPushButton { color: #000000; background-color: #ffffff }</string>
              </property>
              <property name="text">
               <string>Câu 25</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCh_c_n_ng">
    <property name="title">
     <string>Chức năng</string>
    </property>
    <addaction name="setting_action"/>
    <addaction name="return_home_action"/>
    <addaction name="quit_action"/>
   </widget>
   <widget class="QMenu" name="menuTh_ng_tin">
    <property name="title">
     <string>Thông tin</string>
    </property>
    <addaction name="about_app_action"/>
    <addaction name="check_update"/>
    <addaction name="app_version_action"/>
    <addaction name="donate_action"/>
   </widget>
   <addaction name="menuCh_c_n_ng"/>
   <addaction name="menuTh_ng_tin"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="setting_action">
   <property name="text">
    <string>Cài đặt</string>
   </property>
  </action>
  <action name="return_home_action">
   <property name="text">
    <string>Trở về màn hình chính</string>
   </property>
  </action>
  <action name="quit_action">
   <property name="text">
    <string>Thoát chương trình</string>
   </property>
  </action>
  <action name="about_app_action">
   <property name="text">
    <string>Về ứng dụng này</string>
   </property>
  </action>
  <action name="check_update">
   <property name="text">
    <string>Kiểm tra cập nhật</string>
   </property>
  </action>
  <action name="donate_action">
   <property name="text">
    <string>Giúp đỡ tác giả</string>
   </property>
  </action>
  <action name="app_version_action">
   <property name="text">
    <string>Thông tin phiên bản</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
